app/helpers/proposals_helper.rb

```diff
@@ -22,6 +22,30 @@ def proposal_type_year(proposal_type)
     proposal_type.year&.split(",")&.map { |year| year }
   end
 
+  def proposal_same_week_as
+    Proposal.all.where(outcome: "approved").map { |pro| [pro.code] }
+  end
+
+  def assigned_dates(location)
+    return "" unless location
```

Comment:
In Ruby if `location` is set to "", this would still return true. Should be more explicit, like `return "" unless location.present?`